src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -118,8 +118,11 @@ private static Person createEditedPerson(ReadOnlyPerson personToEdit,
         Deadline updatedDeadline = editPersonDescriptor.getDeadline().orElse(personToEdit.getDeadline());
         Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
 
-        return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedPostalCode,
+        Person editedPerson = new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedPostalCode,
                 updatedDebt, updatedInterest, updatedDeadline, updatedTags);
+        editedPerson.setIsBlacklisted(personToEdit.getIsBlacklisted());
```

Comment:
just to clarify, this is just to make sure if the person was originally blacklisted, he would remain so after the edit?

Reply:
Yes exactly!